agent/bench-scripts/pbench-fio

```diff
@@ -92,7 +92,7 @@ function usage {
 	printf "\t\tone or more block sizes in KiB\n"
 	printf "\n"
 	printf -- "\t-s int[,int...] --file-size=int[,int...] (no default)\n"
-	printf "\t\tfile sizes in MiB\n"
+	printf "\t\tfile sizes in bytes\n"
```

Comment:
This is a bit more convoluted that just "bytes."  The parameter is passed along as the `size` field, see https://fio.readthedocs.io/en/latest/fio_doc.html#cmdoption-arg-size.  This means it can take values like, `12`, `12b`, `12k`, `12m`, `12g`, etc.

Reply:
Thanks, Pete; I was trying to figure out exactly which `fio` parameter that connects to, and you wrote this before I succeeded (or wasted too much time trying). 😁

Reply:
So, the fio docs have [an entire section](https://fio.readthedocs.io/en/latest/fio_doc.html#int) explaining how to specify this value.  How much (if any) of that do we want to reproduce here?  Or, do we want to place a reference to that, here?  Or, is the text as I've provided actually sufficient to our needs?  (E.g., just because the value is required to be "in bytes" doesn't mean you cannot provide `12k`, which would mean `12288` bytes.)

Reply:
Perhaps something like:

```
printf "\t\tfile sizes in bytes (accepts modifiers, e.g. '128m', '1500k', etc.)\n"
printf "\t\t(see https://fio.readthedocs.io/en/latest/fio_doc.html#cmdoption-arg-size)\n"
```

Reply:
Done, although I used the `#int` section in the cross-reference, since that's what actually describes how to specify the value (and because it was hard for me to find).

Reply:
GAH!

I'm now getting a new gold file mismatch:
```
line 476: fio: command not found
```
...because this suggestion added a line to the script, the "command not found" is now produced by line **_477_**.  😝

(I'll update the gold file... 😬)